Improve instance type for OMK

eternaltyro opened the pull request:
- Improve instance type to remove disk capacity constraint
- Add tags to all resources (introduces new attributes)
- Add EBS volumes and corresponding device names
- Modify cloudinit script to reflect new device names for mounting and formatting

Note: EBS device names vary depending on the AMI and the volume type.

eternaltyro committed:
Improve instance type for OMK

- Improve instance type to remove disk capacity constraint
- Add tags to all resources (introduces new attributes)
- Add EBS volumes and corresponding device names
- Modify cloudinit script to reflect new device names for mounting and formatting

Note: EBS device names vary depending on the AMI and the volume type.

dakotabenjamin commented on cloudformation/OpenMapKitServer.template.js:
```diff
@@ -100,15 +132,28 @@ const Resources = {
+              Encrypted: false,
+              Iops: 3000,
+              Throughput: 125,
+              VolumeSize: 100,
```
I don't think its necessary for all OMK servers to have a large data directory by default, and doing so will incur unnecessary costs. this should work as a stop-gap measure for the tz server but I'm not sure this is the best solution yet to merge to master.

eternaltyro replied:
I can parameterize VolumeSize and we can customize the value every run.

eternaltyro committed:
Parameterize VolumeSize

eternaltyro committed:
Fix device disk naming

eternaltyro committed:
Fix volume device naming - again

- Ubuntu 16.04 does not follow the device naming conventions of AWS
  So what was expected to be /dev/sdb was actually /dev/nvme1n1

  Therefore the value has been hardcoded for now

eternaltyro committed:
Fix device name for AWS - again - again

dakotabenjamin committed:
Fix missing bracket
